07_cmd_lab/cmdlab.txt

```diff
@@ -3,11 +3,12 @@ Questions online: https://www.tocode.co.il/bundles/unix1/lessons/cmdlab
 
 1. Login to your UNIX account. How do you find:
   - your current shell?
-  - your default shell?
-  - a list of all shells currently installed on the machine?
+ echo $0 - your default shell?
+  echo $SHELL  - a list of all shells currently installed on the machine?  cat /etc/shells       
 
-2. View the file /etc/passwd on your machine with the command "cat /etc/passwd". Then read about the file in "man 5 passwd". What information is held in the file? What are the fields in each line?
+2. View the file /etc/passwd on your machine with the command "cat /etc/passwd". Then read about the file in "man 5 passwd". What information is held in the file? details about the currnt user name passoword      What are the fields in each line?   login name, optional encrypted password, numerical user ID , numerical group ID, user name or comment field, user name directory, optional user command interpreter      
 
-3. In Unix, the /etc folder holds configuration files. Display all files in "/etc" folder ordered by file size (Hint: search in "man ls")
+3. In Unix, the /etc folder holds configuration files. Display all files in "/etc" folder ordered by file size (Hint: search in "man ls")   etc SIZE       
 
 4. Print current date/time formatted as: "Sun 21/03/2010, 14:30". Hint: read "man date" and search for the correct format string.
+   date -I -R   
```

Comment:
Your command yields the following result (on my machine): `date: multiple output formats specified`

The correct answer provides a full format such as: `date '+%a %d/%m/%Y, %H:%M'`